well-known-bots.json

```diff
@@ -7310,6 +7999,7 @@
     ],
     "pattern": "Vercelbot",
     "addition_date": "2024/08/30",
+    "verification": [],
```

Comment:
Please create a follow up issue to ask vercel about verification of their bots

Reply:
https://github.com/arcjet/well-known-bots/issues/24